src/connectMultireducer.js

```diff
@@ -28,22 +30,15 @@ export default function connectMultireducer(mapStateToProps, actions = {}) {
       generateConnectedComponent({multireducerKey}) {
         this.ConnectedComponent =
           connect(
-            (state, ownProps) => {
-              const multireducerKeys = Object.keys(state.multireducer);
-              if (!multireducerKeys.filter(key => key === multireducerKey)[0]) {
-                throw new Error(`No state for multireducer key "${multireducerKey}". You initialized multireducer with "${multireducerKeys.join(', ')}".`);
-              }
-              const slice = state.multireducer[multireducerKey];
-              return mapStateToProps ? mapStateToProps(slice, ownProps) : slice;
-            },
-            multireducerBind(actions, multireducerKey)
+            wrapMapStateToProps(mapStateToProps, multireducerKey),
+            wrapMapDispatchToProps(mapDispatchToProps, multireducerKey),
+            ...rest
```

Comment:
@erikras Oh, sorry if described it not clear. But yes you need to use that `key` to access state slice. Yeah, it's not that convenient as in previous version, but at the other hand you have access to app state.